lib/generators/atomic_cms/scaffold/scaffold_generator.rb

```diff
@@ -0,0 +1,68 @@
+require 'active_support/core_ext/string'
+
+module AtomicCms
+  module Generators
+    class ScaffoldGenerator < Rails::Generators::Base
+      argument :model_name, required: true
+
+      desc <<-DESC.strip_heredoc
+        Create a model, controller, view, admin resource, and route.
+      DESC
+
+      source_root File.expand_path("../../templates", __FILE__)
+
+      def install_model_template
+        setup_scaffold
+        create_template({
+          template_name: 'model.erb',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.